### DISCUSSION TITLE
Location for policy decsription

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/tractusx-edc#992] [General]
Hello everybody,

There is already a PR #856 (right now closed -> started last year) which provides a “full” policy description with several examples (not yet consistent because the policy examples are just collections from other repositories)

In my mind, the description is really helpful and useful, but right now, we are not sure, where it should belong to

- tracusx-edc repository
- connector kit section in github.io repository
- ...

I don't want to lose this information!

What is your opinion?

thx

@arnoweiss @wolf4ood @stefan-ettl @mhellmeier @BenediktSR @florianrusch-zf @matgnt

**Comment by arnoweiss** [comment]
I'd like to see this merged into the Mgmt-API-Walkthrough [1]. I have some time next week to work on this.

[1] https://github.com/eclipse-tractusx/tractusx-edc/blob/1780d4e85723c20274a75a0c7bba92bd93c60728/docs/kit/Development%20View/01_MGMT_API_Walkthrough/02_policies.md

**Reply by florianrusch-zf** [reply]
same opinion

**Comment by matgnt** [comment]
Beware, that there is also lots of examples in the KITS.
We're also working on an Catena-X ODRL Profile which is absolutely required in my opinion. This is still work in progress. You can get an impression here:
https://confluence.catena-x.net/display/PL/Data+Sovereignty+MVP+Release+24.05#DataSovereigntyMVPRelease24.05-KITStoUpdateProposal:

**Reply by stephanbcbauer** [reply]
The initial idea of this documentation (this discussion and #856 ) was to provide a 120% documentation :), are we doing redundant work right now? What i heard ... we definitely need a policy description, examples aso.

**Reply by BenediktSR** [reply]
Hi guys,
Yes, I've seen it too. There is also very detailed documentation in the upcoming Traceability KIT documentation:
[https://eclipse-tractusx.github.io/docs-kits/next/kits/Traceability%20Kit/Software%20Development%20View/Data%20Provider%20Development%20View%20Traceability%20Kit](url)
My PR is also just a collection of the examples I found in the various repositories. 
For me, it would be important to have **one central location** where the information on the policies can be found. As work is currently continuing on the policies, all examples will otherwise have to be changed in all repositories.
